src/olympia/files/tests/test_models.py

```diff
@@ -257,7 +257,7 @@ def test_generate_filename_platform_specific(self):
 
     def test_generate_filename_many_apps(self):
         f = File.objects.get(id=67442)
-        f.version.compatible_apps = (amo.FIREFOX, amo.THUNDERBIRD)
+        f.version.compatible_apps = (amo.THUNDERBIRD, amo.FIREFOX)
```

Comment:
I was a bit scared to see if we use `generate_filename` somewhere to find files by filename instead of using our saved filenames but that doesn't appear to be the case. So we should be good to go here too.